e2e/Makefile

```diff
@@ -83,4 +83,4 @@ notice:
 
 .PHONY: sync-integrations
 sync-integrations:
-	OP_LOG_LEVEL=${LOG_LEVEL} ./op sync integrations --delete
+	OP_LOG_LEVEL=${LOG_LEVEL} ./op sync integrations --delete --remote "elastic:6.8"
```

Comment:
```suggestion
	OP_LOG_LEVEL=${LOG_LEVEL} ./op sync integrations --delete --remote "elastic:7.x"
```

Reply:
We will have to try, some services were not supported in 6.8, and some others required changes for newer versions, we can try to make a run with a 7.x version, and remove the suites that don't pass in 6.8.

I would use a tag or changeset instead of a branch, files in 7.x can still change, and this can affect the list of suites that pass with these files in 6.8.